app/controllers/games_controller.rb

```diff
@@ -22,7 +22,7 @@ def update
 
   def show
     @game = Game.includes(:pieces).find(params[:id])
-    @piece = Piece.find(params[:id])
+    #@piece = Piece.find(params[:id])
   end
```

Comment:
Aleks, can you try the drop and recreate database thing. from the error page you showed it said can't find id.  the other day i got same error for can't find email.  something was wrong with my local database, so might as well try before taking this out completely?